extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -742,6 +749,20 @@ export class AmpStoryPage extends AMP.BaseElement {
         eventInit);
   }
 
+  /**
+  * Checks if the page has any audio
+  * @private
+  */
+  checkPageHasAudio_() {
+    const hasBGPageAudio = this.element.hasAttribute('background-audio');
+    const videos = this.getAllVideos_();
+    const hasVideoWithAudio = videos.find(function(video) {
+      return video.hasAttribute('noaudio') == false;
```

Comment:
This is where I'm having the problem described in my previous comment. For some reason even videos with a noaudio attribute show up as not having it, making every page use the 'sound on' message on unmute. I tested with the debuggers and when I do video.getAttributesByName(), the noaudio attribute won't show up.  Any insight to why this might be happening would be really helpful! @gmajoulet